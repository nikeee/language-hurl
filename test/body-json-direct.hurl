POST https://example.org/InStock
{
  "a": 1
}
HTTP 200
