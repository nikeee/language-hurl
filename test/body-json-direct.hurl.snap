>POST https://example.org/InStock
#^^^^ source.hurl keyword.control.method.hurl
#    ^ source.hurl
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.unquoted.url.hurl string.unquoted.url.hurl
>{
#^^ source.hurl
>  "a": 1
#^^ source.hurl
#  ^^^ source.hurl string.quoted.double.hurl
#     ^^^^ source.hurl
>}
#^^ source.hurl
>HTTP 200
#^^^^ source.hurl constant.language.version.hurl
#    ^^^^ source.hurl constant.numeric.status.hurl
>